File: openapi-generator/javascript-closure-angular/API/Client/InlineResponse2002.js
goog.provide('API.Client.inline_response_200_2');

/**
 * @record
 */
API.Client.InlineResponse2002 = function() {}

/**
 * @type {!API.Client.Pagination}
 * @export
 */
API.Client.InlineResponse2002.prototype.pagination;

/**
 * Group ID to query.
 * @type {!number}
 * @export
 */
API.Client.InlineResponse2002.prototype.groupId;

/**
 * @type {!Array<!API.Client.Vehicle>}
 * @export
 */
API.Client.InlineResponse2002.prototype.vehicles;

File: openapi-generator/scalaz/src/main/scala/org/openapitools/client/api/InlineResponse2002.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import InlineResponse2002._

case class InlineResponse2002 (
  pagination: Option[Pagination],
/* Group ID to query. */
  groupId: Option[Long],
vehicles: Option[List[Vehicle]])

object InlineResponse2002 {
  import DateTimeCodecs._

  implicit val InlineResponse2002CodecJson: CodecJson[InlineResponse2002] = CodecJson.derive[InlineResponse2002]
  implicit val InlineResponse2002Decoder: EntityDecoder[InlineResponse2002] = jsonOf[InlineResponse2002]
  implicit val InlineResponse2002Encoder: EntityEncoder[InlineResponse2002] = jsonEncoderOf[InlineResponse2002]
}
